services/QuillLMS/client/app/bundles/Diagnostic/components/questions/responseComponent.jsx

```diff
@@ -72,6 +72,8 @@ class ResponseComponent extends React.Component {
     } else if (this.props.states[this.props.questionID] === C.SHOULD_RELOAD_RESPONSES && prevProps.states[prevProps.questionID] !== C.SHOULD_RELOAD_RESPONSES) {
       this.props.dispatch(questionActions.clearQuestionState(this.props.questionID));
       this.searchResponses();
+    } else if (!_.isEqual(this.props.filters.responses, prevProps.filters.responses)) {
+      this.setState({responses: this.props.filters.responses})
```

Comment:
Even though this file isn't up to this standard in general, I still like to go with the linter standards on new code  (like extracting `{ filters, }` from `this.props` and using that in these lines) because it does make it easier to read.